### DISCUSSION TITLE
How to get access tokens for users signed in via Github login asyncrhonously from an edge function?

### DISCUSSION THREAD

**Discussion by tgirotto** [supabase/supabase#28763] [Questions]
My app uses Github login and runs some cronjobs (eg clean up their repositories) on behalf of my users once a day. How can I get an access token server side and perform an action on their behalf in my cronjob?

**Comment by GaryAustin1** [comment]
You can just use service_role and pass the user id to your function and use filters.
You can mint a jwt in the client with the user id, the jwt secret and a jwt library.
